package-lock.json

```diff
@@ -9,42 +9,39 @@
       "version": "0.1.0",
       "dependencies": {
         "@types/node": "20.6.3",
-        "@types/react": "18.2.22",
-        "@types/react-dom": "18.2.7",
+        "@types/react": "^18.3.3",
+        "@types/react-dom": "^18.3.0",
         "autoprefixer": "10.4.15",
         "cheerio": "^1.0.0-rc.12",
-        "eslint": "8.49.0",
-        "eslint-config-next": "13.5.2",
-        "next": "13.5.2",
+        "next": "^14.2.4",
         "postcss": "8.4.30",
-        "react": "18.2.0",
-        "react-dom": "18.2.0",
-        "sass": "^1.77.1",
+        "react": "^18.3.1",
+        "react-dom": "^18.3.1",
+        "sass": "^1.77.6",
         "tailwindcss": "3.3.3",
         "typescript": "5.2.2"
       },
       "devDependencies": {
+        "@eslint/js": "^9.2.0",
+        "@stylistic/eslint-plugin-js": "^2.1.0",
         "@testing-library/jest-dom": "^6.1.4",
         "@testing-library/react": "^14.1.2",
+        "eslint": "^8.57.0",
```

Comment:
I know I am to blame here as much as anyone, but I am just now seeing that version 9.x.x of eslint has been out for ~5mo. Is it worth at least putting an issue in this milestone, so that we can be kept up to date.

Reply:
Yeah I think a follow up ticket should go in this milestone